src/Arcus.Security.Core/Secret.cs

```diff
@@ -34,5 +36,10 @@ public Secret(string value, string version)
         /// </summary>
         /// <remarks>Version is not checked for <c>null</c>.</remarks>
         public string Version { get; }
+
+        /// <summary>
+        /// Gets the optional expiration date of the secret.
```

Comment:
Since it's nullable it's clear that it's optional, no?
```suggestion
        /// Gets the expiration date of the secret.
```